### ISSUE TITLE
$meteor.object lack of helpers

### ISSUE THREAD

**Issue by goooseman** [Urigo/angular-meteor#276]
I'm using meteor with [dburles:collection-helpers](https://atmospherejs.com/dburles/collection-helpers).
For example, I have a collection Companies and a helper 'fullName'.

Everything works fine with $meteor.collection. When I iterate through companies with ``` ng-repeat="company in companies" ``` I can easily do ``` {{ company.fullName() }} ```

But if I use ``` $meteor.object(Companies, $stateParams.companyId) ```, the object doesn't have a fullName() function. 

The strangest fact, that when I use the old API ( ``` $collection(Companies).bindOne($scope, 'company', $stateParams.companyId) ``` ), everything works fine: I can call ``` $scope.company.fullName() ```.

**Comment by netanelgilad** [comment]
We had a similar issue with collections but that was resolved when we moved from observeChanges to observe.
Here the situation is quite different. This bug comes from that $meteorObject copies the properties of the object it gets from the collection and doesn't take the object itself. That is why the function that are on the prototype (like helpers) are not found on the object returned by $meteorObject.
The reason why this copying is done is that changes from the server would propogate to places holding a reference to the object. If we would replace the object totally then whoever holds the old reference will never see the updates.
We could somehow get the prototype of the object from the collection and set that as the prototype of the object we are returning, then see if that solves the case.
@Urigo any ideas?
